### PR TITLE
docs(aws): add config examples

### PR DESCRIPTION
**Pull request by robvadai**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.

## Changes:

Example for AWS profile set up

### CODE REVIEW

**Review by maksyms — CHANGES_REQUESTED** [review]
No comments apart from the below suggestions, tbh. 👍

File: plugins/aws/README.md [hunk 1]
```diff
@@ -43,3 +43,31 @@ the current `$AWS_PROFILE`. It uses two variables to control how that is shown:
 * ZSH_THEME_AWS_PREFIX: sets the prefix of the AWS_PROFILE. Defaults to `<aws:`.
 
 * ZSH_THEME_AWS_SUFFIX: sets the suffix of the AWS_PROFILE. Defaults to `>`.
+
+## Configuration
+
```
[the remark this answers is not in the document]
Comment (robvadai) [reply plugins/aws/README.md:48]: Added link, please verify it

**Review by maksyms — APPROVED** [review]

**Comment by maksyms** [comment]
@mcornella This is documentation-only changes, making it clearer how to configure AWS CLI. Good to merge.

**Comment by robvadai** [comment]
Appreciated guys, thank you

**Pull request merged**